src/index.ts

```diff
@@ -159,11 +164,43 @@ export class MondayRedisService extends BaseConnector {
               this.serialize(item[title], title),
             )
           ),
-          this.redis.hset(this.namesKey, item.name, item.id)
+          this.redis.hset(this.namesKey, item.name, item.id),
         ]))
-        .flat()
+          .flat()
       )
     }
+
+    this.boardId = await this.getBoardId()
+    this.monday.on(
+      {
+        type: 'ChangeColumnValue',
+        boardId: this.boardId,
+      },
+      async (event) => {
+        const { boardId, itemId, columnTitle, value: { value } } = event
+        if (this.boardId === parseInt(boardId, 10)) {
+          const redisCurrentValue = await this.redis.hget(
```

Comment:
This is not atomic, so I'm not sure if it's super valuable. You might as well just update Redis and destage, no?

Reply:
I did that for optimisation. When you update inventory via Monday sdk, you get the event triggered which trigger a change and so on. By checking if I have the same value, I stop the loop.

Reply:
Ah. So not optimization - but necessity. This is much tricker in case changes come in between the write to Monday and the webhook, and I don't think the current code solves for that.